scripts/deploy-daofactory.js

```diff
@@ -0,0 +1,32 @@
+const globalArtifacts = this.artifacts // Not injected unless called directly via truffle
```

Comment:
Using `this.artifacts` is a bit of a hack; not sure how this is injected by truffle but `global.artifacts` doesn't work :(.